app/views/registration/steps/_details.html.haml

```diff
@@ -57,6 +56,24 @@
                 %select.chunky{ id: 'enterprise_country', name: 'country', required: true, ng: { init: "setDefaultCountry(#{Spree::Config[:default_country_id]})", model: 'enterprise.country', options: 'c as c.name for c in countries' } }
                 %span.error{ ng: { show: "details.country.$error.required && submitted" } }
                   = t(".country_field_error")
+      - if using_google_maps?
```

Comment:
I know I suggested this but now I think it would have been easier to read in a separate partial:
```
render `location_map` if using_google_maps?
```

It's okay as is as well.